requirements/test.txt

```diff
@@ -1,7 +1,9 @@
 -r base.txt
 pytest==6.2.4
 pytest-cov==2.12.0
+pytest-factoryboy==2.1.0
 pytest-isort==2.0.0
+pytest-factoryboy==2.1.0
```

Comment:
^ this seems to be duplicate entry

Reply:
@thelostone-mc Nice catch. I'll fix that